Compilation error with MY_RFM69_ENABLE_LISTENMODE

Hi, just out of curiosity I've enabled MY_RFM69_ENABLE_LISTENMODE in v2.2.0 but I got the following compilation error. Looks like RFM69_setListenConfig is undefined, not sure if this is expected or if the user has to define somehow this function. Not critical of course but just in case there is something missing somewhere. Thanks

> C:\Data\PortableApps\arduino\arduino-builder -dump-prefs -logger=machine -hardware C:\Data\PortableApps\arduino\hardware -hardware C:\Data\PortableApps\arduino\portable\packages -hardware C:\Data\PortableApps\arduino\local\hardware -tools C:\Data\PortableApps\arduino\tools-builder -tools C:\Data\PortableApps\arduino\hardware\tools\avr -tools C:\Data\PortableApps\arduino\portable\packages -built-in-libraries C:\Data\PortableApps\arduino\libraries -libraries C:\Data\PortableApps\arduino\local\libraries -fqbn=ProMiniLP:avr:ProMiniLP:cpu=1MhzIC_18v -ide-version=10801 -build-path C:\Data\TempForPortableApps\arduino_build_893676 -warnings=more -prefs=build.warn_data_percentage=75 -prefs=runtime.tools.arduinoOTA.path=C:\Data\PortableApps\arduino\portable\packages\arduino\tools\arduinoOTA\1.1.1 -prefs=runtime.tools.avrdude.path=C:\Data\PortableApps\arduino\portable\packages\arduino\tools\avrdude\6.3.0-arduino9 -prefs=runtime.tools.avr-gcc.path=C:\Data\PortableApps\arduino\portable\packages\arduino\tools\avr-gcc\4.9.2-atmel3.5.4-arduino2 -verbose P:\Personal\Projects\MySensors\mike\gateway\gateway.ino
> C:\Data\PortableApps\arduino\arduino-builder -compile -logger=machine -hardware C:\Data\PortableApps\arduino\hardware -hardware C:\Data\PortableApps\arduino\portable\packages -hardware C:\Data\PortableApps\arduino\local\hardware -tools C:\Data\PortableApps\arduino\tools-builder -tools C:\Data\PortableApps\arduino\hardware\tools\avr -tools C:\Data\PortableApps\arduino\portable\packages -built-in-libraries C:\Data\PortableApps\arduino\libraries -libraries C:\Data\PortableApps\arduino\local\libraries -fqbn=ProMiniLP:avr:ProMiniLP:cpu=1MhzIC_18v -ide-version=10801 -build-path C:\Data\TempForPortableApps\arduino_build_893676 -warnings=more -prefs=build.warn_data_percentage=75 -prefs=runtime.tools.arduinoOTA.path=C:\Data\PortableApps\arduino\portable\packages\arduino\tools\arduinoOTA\1.1.1 -prefs=runtime.tools.avrdude.path=C:\Data\PortableApps\arduino\portable\packages\arduino\tools\avrdude\6.3.0-arduino9 -prefs=runtime.tools.avr-gcc.path=C:\Data\PortableApps\arduino\portable\packages\arduino\tools\avr-gcc\4.9.2-atmel3.5.4-arduino2 -verbose P:\Personal\Projects\MySensors\mike\gateway\gateway.ino
> Using board 'ProMiniLP' from platform in folder: C:\Data\PortableApps\arduino\local\hardware\ProMiniLP\avr
> Using core 'arduino' from platform in folder: C:\Data\PortableApps\arduino\portable\packages\arduino\hardware\avr\1.6.21
> WARNING: Category 'Sensor' in library Adafruit CCS811 Library is not valid. Setting to 'Uncategorized'
> WARNING: Spurious .ci folder in 'MySensors' library
> WARNING: Spurious .mystools folder in 'MySensors' library
> Detecting libraries used...
> "C:\Data\PortableApps\arduino\portable\packages\arduino\tools\avr-gcc\4.9.2-atmel3.5.4-arduino2/bin/avr-g++" -c -g -Os -w -std=gnu++11 -fpermissive -fno-exceptions -ffunction-sections -fdata-sections -fno-threadsafe-statics  -flto -w -x c++ -E -CC -mmcu=atmega328p -DF_CPU=1000000L -DARDUINO=10801 -DARDUINO_AVR_PRO -DARDUINO_ARCH_AVR   "-IC:\Data\PortableApps\arduino\portable\packages\arduino\hardware\avr\1.6.21\cores\arduino" "-IC:\Data\PortableApps\arduino\local\hardware\ProMiniLP\avr\variants\prominilp" "C:\Data\TempForPortableApps\arduino_build_893676\sketch\gateway.ino.cpp" -o "nul"
> "C:\Data\PortableApps\arduino\portable\packages\arduino\tools\avr-gcc\4.9.2-atmel3.5.4-arduino2/bin/avr-g++" -c -g -Os -w -std=gnu++11 -fpermissive -fno-exceptions -ffunction-sections -fdata-sections -fno-threadsafe-statics  -flto -w -x c++ -E -CC -mmcu=atmega328p -DF_CPU=1000000L -DARDUINO=10801 -DARDUINO_AVR_PRO -DARDUINO_ARCH_AVR   "-IC:\Data\PortableApps\arduino\portable\packages\arduino\hardware\avr\1.6.21\cores\arduino" "-IC:\Data\PortableApps\arduino\local\hardware\ProMiniLP\avr\variants\prominilp" "-IC:\Data\PortableApps\arduino\local\libraries\MySensors-2.2.0" "C:\Data\TempForPortableApps\arduino_build_893676\sketch\gateway.ino.cpp" -o "nul"
> "C:\Data\PortableApps\arduino\portable\packages\arduino\tools\avr-gcc\4.9.2-atmel3.5.4-arduino2/bin/avr-g++" -c -g -Os -w -std=gnu++11 -fpermissive -fno-exceptions -ffunction-sections -fdata-sections -fno-threadsafe-statics  -flto -w -x c++ -E -CC -mmcu=atmega328p -DF_CPU=1000000L -DARDUINO=10801 -DARDUINO_AVR_PRO -DARDUINO_ARCH_AVR   "-IC:\Data\PortableApps\arduino\portable\packages\arduino\hardware\avr\1.6.21\cores\arduino" "-IC:\Data\PortableApps\arduino\local\hardware\ProMiniLP\avr\variants\prominilp" "-IC:\Data\PortableApps\arduino\local\libraries\MySensors-2.2.0" "-IC:\Data\PortableApps\arduino\portable\packages\arduino\hardware\avr\1.6.21\libraries\SPI\src" "C:\Data\TempForPortableApps\arduino_build_893676\sketch\gateway.ino.cpp" -o "nul"
> "C:\Data\PortableApps\arduino\portable\packages\arduino\tools\avr-gcc\4.9.2-atmel3.5.4-arduino2/bin/avr-g++" -c -g -Os -w -std=gnu++11 -fpermissive -fno-exceptions -ffunction-sections -fdata-sections -fno-threadsafe-statics  -flto -w -x c++ -E -CC -mmcu=atmega328p -DF_CPU=1000000L -DARDUINO=10801 -DARDUINO_AVR_PRO -DARDUINO_ARCH_AVR   "-IC:\Data\PortableApps\arduino\portable\packages\arduino\hardware\avr\1.6.21\cores\arduino" "-IC:\Data\PortableApps\arduino\local\hardware\ProMiniLP\avr\variants\prominilp" "-IC:\Data\PortableApps\arduino\local\libraries\MySensors-2.2.0" "-IC:\Data\PortableApps\arduino\portable\packages\arduino\hardware\avr\1.6.21\libraries\SPI\src" "-IC:\Data\PortableApps\arduino\portable\packages\arduino\hardware\avr\1.6.21\libraries\Wire\src" "C:\Data\TempForPortableApps\arduino_build_893676\sketch\gateway.ino.cpp" -o "nul"
> "C:\Data\PortableApps\arduino\portable\packages\arduino\tools\avr-gcc\4.9.2-atmel3.5.4-arduino2/bin/avr-g++" -c -g -Os -w -std=gnu++11 -fpermissive -fno-exceptions -ffunction-sections -fdata-sections -fno-threadsafe-statics  -flto -w -x c++ -E -CC -mmcu=atmega328p -DF_CPU=1000000L -DARDUINO=10801 -DARDUINO_AVR_PRO -DARDUINO_ARCH_AVR   "-IC:\Data\PortableApps\arduino\portable\packages\arduino\hardware\avr\1.6.21\cores\arduino" "-IC:\Data\PortableApps\arduino\local\hardware\ProMiniLP\avr\variants\prominilp" "-IC:\Data\PortableApps\arduino\local\libraries\MySensors-2.2.0" "-IC:\Data\PortableApps\arduino\portable\packages\arduino\hardware\avr\1.6.21\libraries\SPI\src" "-IC:\Data\PortableApps\arduino\portable\packages\arduino\hardware\avr\1.6.21\libraries\Wire\src" "-IC:\Data\PortableApps\arduino\local\libraries\Sodaq_SHT2x\src" "C:\Data\TempForPortableApps\arduino_build_893676\sketch\gateway.ino.cpp" -o "nul"
> "C:\Data\PortableApps\arduino\portable\packages\arduino\tools\avr-gcc\4.9.2-atmel3.5.4-arduino2/bin/avr-g++" -c -g -Os -w -std=gnu++11 -fpermissive -fno-exceptions -ffunction-sections -fdata-sections -fno-threadsafe-statics  -flto -w -x c++ -E -CC -mmcu=atmega328p -DF_CPU=1000000L -DARDUINO=10801 -DARDUINO_AVR_PRO -DARDUINO_ARCH_AVR   "-IC:\Data\PortableApps\arduino\portable\packages\arduino\hardware\avr\1.6.21\cores\arduino" "-IC:\Data\PortableApps\arduino\local\hardware\ProMiniLP\avr\variants\prominilp" "-IC:\Data\PortableApps\arduino\local\libraries\MySensors-2.2.0" "-IC:\Data\PortableApps\arduino\portable\packages\arduino\hardware\avr\1.6.21\libraries\SPI\src" "-IC:\Data\PortableApps\arduino\portable\packages\arduino\hardware\avr\1.6.21\libraries\Wire\src" "-IC:\Data\PortableApps\arduino\local\libraries\Sodaq_SHT2x\src" "C:\Data\PortableApps\arduino\portable\packages\arduino\hardware\avr\1.6.21\libraries\SPI\src\SPI.cpp" -o "nul"
> "C:\Data\PortableApps\arduino\portable\packages\arduino\tools\avr-gcc\4.9.2-atmel3.5.4-arduino2/bin/avr-g++" -c -g -Os -w -std=gnu++11 -fpermissive -fno-exceptions -ffunction-sections -fdata-sections -fno-threadsafe-statics  -flto -w -x c++ -E -CC -mmcu=atmega328p -DF_CPU=1000000L -DARDUINO=10801 -DARDUINO_AVR_PRO -DARDUINO_ARCH_AVR   "-IC:\Data\PortableApps\arduino\portable\packages\arduino\hardware\avr\1.6.21\cores\arduino" "-IC:\Data\PortableApps\arduino\local\hardware\ProMiniLP\avr\variants\prominilp" "-IC:\Data\PortableApps\arduino\local\libraries\MySensors-2.2.0" "-IC:\Data\PortableApps\arduino\portable\packages\arduino\hardware\avr\1.6.21\libraries\SPI\src" "-IC:\Data\PortableApps\arduino\portable\packages\arduino\hardware\avr\1.6.21\libraries\Wire\src" "-IC:\Data\PortableApps\arduino\local\libraries\Sodaq_SHT2x\src" "C:\Data\PortableApps\arduino\portable\packages\arduino\hardware\avr\1.6.21\libraries\Wire\src\Wire.cpp" -o "nul"
> "C:\Data\PortableApps\arduino\portable\packages\arduino\tools\avr-gcc\4.9.2-atmel3.5.4-arduino2/bin/avr-g++" -c -g -Os -w -std=gnu++11 -fpermissive -fno-exceptions -ffunction-sections -fdata-sections -fno-threadsafe-statics  -flto -w -x c++ -E -CC -mmcu=atmega328p -DF_CPU=1000000L -DARDUINO=10801 -DARDUINO_AVR_PRO -DARDUINO_ARCH_AVR   "-IC:\Data\PortableApps\arduino\portable\packages\arduino\hardware\avr\1.6.21\cores\arduino" "-IC:\Data\PortableApps\arduino\local\hardware\ProMiniLP\avr\variants\prominilp" "-IC:\Data\PortableApps\arduino\local\libraries\MySensors-2.2.0" "-IC:\Data\PortableApps\arduino\portable\packages\arduino\hardware\avr\1.6.21\libraries\SPI\src" "-IC:\Data\PortableApps\arduino\portable\packages\arduino\hardware\avr\1.6.21\libraries\Wire\src" "-IC:\Data\PortableApps\arduino\local\libraries\Sodaq_SHT2x\src" "C:\Data\PortableApps\arduino\portable\packages\arduino\hardware\avr\1.6.21\libraries\Wire\src\utility\twi.c" -o "nul"
> "C:\Data\PortableApps\arduino\portable\packages\arduino\tools\avr-gcc\4.9.2-atmel3.5.4-arduino2/bin/avr-g++" -c -g -Os -w -std=gnu++11 -fpermissive -fno-exceptions -ffunction-sections -fdata-sections -fno-threadsafe-statics  -flto -w -x c++ -E -CC -mmcu=atmega328p -DF_CPU=1000000L -DARDUINO=10801 -DARDUINO_AVR_PRO -DARDUINO_ARCH_AVR   "-IC:\Data\PortableApps\arduino\portable\packages\arduino\hardware\avr\1.6.21\cores\arduino" "-IC:\Data\PortableApps\arduino\local\hardware\ProMiniLP\avr\variants\prominilp" "-IC:\Data\PortableApps\arduino\local\libraries\MySensors-2.2.0" "-IC:\Data\PortableApps\arduino\portable\packages\arduino\hardware\avr\1.6.21\libraries\SPI\src" "-IC:\Data\PortableApps\arduino\portable\packages\arduino\hardware\avr\1.6.21\libraries\Wire\src" "-IC:\Data\PortableApps\arduino\local\libraries\Sodaq_SHT2x\src" "C:\Data\PortableApps\arduino\local\libraries\Sodaq_SHT2x\src\Sodaq_SHT2x.cpp" -o "nul"
> Generating function prototypes...
> "C:\Data\PortableApps\arduino\portable\packages\arduino\tools\avr-gcc\4.9.2-atmel3.5.4-arduino2/bin/avr-g++" -c -g -Os -w -std=gnu++11 -fpermissive -fno-exceptions -ffunction-sections -fdata-sections -fno-threadsafe-statics  -flto -w -x c++ -E -CC -mmcu=atmega328p -DF_CPU=1000000L -DARDUINO=10801 -DARDUINO_AVR_PRO -DARDUINO_ARCH_AVR   "-IC:\Data\PortableApps\arduino\portable\packages\arduino\hardware\avr\1.6.21\cores\arduino" "-IC:\Data\PortableApps\arduino\local\hardware\ProMiniLP\avr\variants\prominilp" "-IC:\Data\PortableApps\arduino\local\libraries\MySensors-2.2.0" "-IC:\Data\PortableApps\arduino\portable\packages\arduino\hardware\avr\1.6.21\libraries\SPI\src" "-IC:\Data\PortableApps\arduino\portable\packages\arduino\hardware\avr\1.6.21\libraries\Wire\src" "-IC:\Data\PortableApps\arduino\local\libraries\Sodaq_SHT2x\src" "C:\Data\TempForPortableApps\arduino_build_893676\sketch\gateway.ino.cpp" -o "C:\Data\TempForPortableApps\arduino_build_893676\preproc\ctags_target_for_gcc_minus_e.cpp"
> "C:\Data\PortableApps\arduino\tools-builder\ctags\5.8-arduino11/ctags" -u --language-force=c++ -f - --c++-kinds=svpf --fields=KSTtzns --line-directives "C:\Data\TempForPortableApps\arduino_build_893676\preproc\ctags_target_for_gcc_minus_e.cpp"
> Compiling sketch...
> "C:\Data\PortableApps\arduino\portable\packages\arduino\tools\avr-gcc\4.9.2-atmel3.5.4-arduino2/bin/avr-g++" -c -g -Os -Wall -std=gnu++11 -fpermissive -fno-exceptions -ffunction-sections -fdata-sections -fno-threadsafe-statics -MMD -flto -mmcu=atmega328p -DF_CPU=1000000L -DARDUINO=10801 -DARDUINO_AVR_PRO -DARDUINO_ARCH_AVR   "-IC:\Data\PortableApps\arduino\portable\packages\arduino\hardware\avr\1.6.21\cores\arduino" "-IC:\Data\PortableApps\arduino\local\hardware\ProMiniLP\avr\variants\prominilp" "-IC:\Data\PortableApps\arduino\local\libraries\MySensors-2.2.0" "-IC:\Data\PortableApps\arduino\portable\packages\arduino\hardware\avr\1.6.21\libraries\SPI\src" "-IC:\Data\PortableApps\arduino\portable\packages\arduino\hardware\avr\1.6.21\libraries\Wire\src" "-IC:\Data\PortableApps\arduino\local\libraries\Sodaq_SHT2x\src" "C:\Data\TempForPortableApps\arduino_build_893676\sketch\gateway.ino.cpp" -o "C:\Data\TempForPortableApps\arduino_build_893676\sketch\gateway.ino.cpp.o"
> In file included from C:\Data\PortableApps\arduino\local\libraries\MySensors-2.2.0/MySensors.h:343:0,
> 
>                  from C:\Data\TempForPortableApps\arduino_build_893676\sketch\NodeManagerLibrary.h:146,
> 
>                  from P:\Personal\Projects\MySensors\mike\gateway\gateway.ino:317:
> 
> C:\Data\PortableApps\arduino\local\libraries\MySensors-2.2.0/drivers/RFM69/new/RFM69_new.cpp: In function 'void RFM69_interruptHandler()':
> 
> C:\Data\PortableApps\arduino\local\libraries\MySensors-2.2.0/drivers/RFM69/new/RFM69_new.cpp:266:5: warning: variable 'burstRemaining' set but not used [-Wunused-but-set-variable]
> 
>    } burstRemaining;
> 
>      ^
> 
> C:\Data\PortableApps\arduino\local\libraries\MySensors-2.2.0/drivers/RFM69/new/RFM69_new.cpp: In function 'bool RFM69_setRadioMode(rfm69_radio_mode_t)':
> 
> C:\Data\PortableApps\arduino\local\libraries\MySensors-2.2.0/drivers/RFM69/new/RFM69_new.cpp:539:45: warning: suggest parentheses around comparison in operand of '&' [-Wparentheses]
> 
>    while (RFM69_readReg(RFM69_REG_IRQFLAGS2) & RFM69_IRQFLAGS2_PACKETSENT ==
> 
>                                              ^
> 
> C:\Data\PortableApps\arduino\local\libraries\MySensors-2.2.0/drivers/RFM69/new/RFM69_new.cpp:568:45: error: 'RFM69_setListenConfig' was not declared in this scope
> 
>                          RFM69_LISTEN1_END_10);
> 
>                                              ^
> 
> C:\Data\PortableApps\arduino\local\libraries\MySensors-2.2.0/drivers/RFM69/new/RFM69_new.cpp:569:41: error: 'RFM69_setListenCoefIdle' was not declared in this scope
> 
>    RFM69_setListenCoefIdle(idleListenCoef);
> 
>                                          ^
> 
> C:\Data\PortableApps\arduino\local\libraries\MySensors-2.2.0/drivers/RFM69/new/RFM69_new.cpp:570:37: error: 'RFM69_setListenCoefRx' was not declared in this scope
> 
>    RFM69_setListenCoefRx(rxListenCoef);
> 
>                                      ^
> 
> C:\Data\PortableApps\arduino\local\libraries\MySensors-2.2.0/drivers/RFM69/new/RFM69_new.cpp: In function 'void RFM69_listenModeApplyHighSpeedSettings()':
> 
> C:\Data\PortableApps\arduino\local\libraries\MySensors-2.2.0/drivers/RFM69/new/RFM69_new.cpp:979:7: error: '_isHighSpeed' was not declared in this scope
> 
>   if (!_isHighSpeed) {
> 
>        ^
> 
> C:\Data\PortableApps\arduino\local\libraries\MySensors-2.2.0/drivers/RFM69/new/RFM69_new.cpp: In function 'bool RFM69_listenModeEnd()':
> 
> C:\Data\PortableApps\arduino\local\libraries\MySensors-2.2.0/drivers/RFM69/new/RFM69_new.cpp:1059:1: warning: no return statement in function returning non-void [-Wreturn-type]
> 
>  }
> 
>  ^
> 
> C:\Data\PortableApps\arduino\local\libraries\MySensors-2.2.0/drivers/RFM69/new/RFM69_new.cpp: In function 'void RFM69_listenModeSendBurst(uint8_t, uint8_t*, uint8_t)':
> 
> C:\Data\PortableApps\arduino\local\libraries\MySensors-2.2.0/drivers/RFM69/new/RFM69_new.cpp:1112:14: error: no match for call to '(SPIClass) (int)'
> 
>              4);      // two bytes for target and sender node, two bytes for the burst time remaining
> 
>               ^
> 
> C:\Data\PortableApps\arduino\local\libraries\MySensors-2.2.0/drivers/RFM69/new/RFM69_new.cpp:1114:22: error: '_address' was not declared in this scope
> 
>    RFM69_SPI.transfer(_address);
> 
>                       ^
> 
> C:\Data\PortableApps\arduino\local\libraries\MySensors-2.2.0/drivers/RFM69/new/RFM69_new.cpp: At global scope:
> 
> C:\Data\PortableApps\arduino\local\libraries\MySensors-2.2.0/drivers/RFM69/new/RFM69_new.cpp:663:12: warning: 'void RFM69_ATCmode(bool, int16_t)' defined but not used [-Wunused-function]
> 
>  LOCAL void RFM69_ATCmode(const bool onOff, const int16_t targetRSSI)
> 
>             ^
> 
> C:\Data\PortableApps\arduino\local\libraries\MySensors-2.2.0/drivers/RFM69/new/RFM69_new.cpp:1005:12: warning: 'void RFM69_ListenModeStart()' defined but not used [-Wunused-function]
> 
>  LOCAL void RFM69_ListenModeStart(void)
> 
>             ^
> 
> C:\Data\PortableApps\arduino\local\libraries\MySensors-2.2.0/drivers/RFM69/new/RFM69_new.cpp:1040:12: warning: 'bool RFM69_listenModeEnd()' defined but not used [-Wunused-function]
> 
>  LOCAL bool RFM69_listenModeEnd(void)
> 
>             ^
> 
> C:\Data\PortableApps\arduino\local\libraries\MySensors-2.2.0/drivers/RFM69/new/RFM69_new.cpp:1072:12: warning: 'void RFM69_listenModeSendBurst(uint8_t, uint8_t*, uint8_t)' defined but not used [-Wunused-function]
> 
>  LOCAL void RFM69_listenModeSendBurst(const uint8_t recipient, uint8_t* data, const uint8_t len)
> 
>             ^
> 
> C:\Data\PortableApps\arduino\local\libraries\MySensors-2.2.0/drivers/RFM69/new/RFM69_new.cpp:1204:12: warning: 'void RFM69_listenModeGetDurations(uint32_t&, uint32_t&)' defined but not used [-Wunused-function]
> 
>  LOCAL void RFM69_listenModeGetDurations(uint32_t &rxDuration, uint32_t &idleDuration)
> 
>             ^
> 
> C:\Data\PortableApps\arduino\local\libraries\MySensors-2.2.0/drivers/RFM69/new/RFM69_new.cpp:917:12: warning: 'void RFM69_setListenConfig(uint8_t, uint8_t, uint8_t, uint8_t)' defined but not used [-Wunused-function]
> 
>  LOCAL void RFM69_setListenConfig(const uint8_t listenResolIdle, const uint8_t listenResolRx,
> 
>             ^
> 
> C:\Data\PortableApps\arduino\local\libraries\MySensors-2.2.0/drivers/RFM69/new/RFM69_new.cpp:955:12: warning: 'void RFM69_setListenCoefIdle(uint8_t)' defined but not used [-Wunused-function]
> 
>  LOCAL void RFM69_setListenCoefIdle(const uint8_t coeffIdle)
> 
>             ^
> 
> C:\Data\PortableApps\arduino\local\libraries\MySensors-2.2.0/drivers/RFM69/new/RFM69_new.cpp:966:12: warning: 'void RFM69_setListenCoefRx(uint8_t)' defined but not used [-Wunused-function]
> 
>  LOCAL void RFM69_setListenCoefRx(const uint8_t coeffRX)
> 
>             ^
> 
> Using library MySensors-2.2.0 at version 2.2.0 in folder: C:\Data\PortableApps\arduino\local\libraries\MySensors-2.2.0 
> Using library SPI at version 1.0 in folder: C:\Data\PortableApps\arduino\portable\packages\arduino\hardware\avr\1.6.21\libraries\SPI 
> Using library Wire at version 1.0 in folder: C:\Data\PortableApps\arduino\portable\packages\arduino\hardware\avr\1.6.21\libraries\Wire 
> Using library Sodaq_SHT2x at version 1.2.0 in folder: C:\Data\PortableApps\arduino\local\libraries\Sodaq_SHT2x 
> exit status 1
> Error compiling for board ProMini LowPower - atmega328p.
>

--- a/MyConfig.h
+++ b/MyConfig.h
@@ -768,22 +768,6 @@
 //#define MY_RFM69_ENABLE_ENCRYPTION
 
 /**
- * @def MY_RFM69_ENABLE_LISTENMODE
- * @brief Define this if you need listenmode, or skip it to save memory
- */
-//#define MY_RFM69_ENABLE_LISTENMODE
-
-#if defined(MY_RFM69_ENABLE_LISTENMODE) && !defined(MY_RFM69_DEFAULT_LISTEN_RX_US)
-// By default, receive for 256uS in listen mode and idle for ~1s
-#define MY_RFM69_DEFAULT_LISTEN_RX_US	(256)
-#endif
-
-#if defined(MY_RFM69_ENABLE_LISTENMODE) && !defined(MY_RFM69_DEFAULT_LISTEN_IDLE_US)
-// By default, receive for 256uS in listen mode and idle for ~1s
-#define  MY_RFM69_DEFAULT_LISTEN_IDLE_US (1*1000000ul)
-#endif
-
-/**
  * @def MY_RFM69_MODEM_CONFIGURATION
  * @brief %RFM69 modem configuration, default is %RFM69_FSK_BR55_5_FD50
  *
@@ -2334,7 +2318,6 @@
 #define MY_RFM69_RST_PIN
 #define MY_DEBUG_VERBOSE_RFM69
 #define MY_DEBUG_VERBOSE_RFM69_REGISTERS
-#define MY_RFM69_ENABLE_LISTENMODE
 // RFM95
 #define MY_RADIO_RFM95
 #define MY_DEBUG_VERBOSE_RFM95

